bases/node-ts.json

```
@@ -4,6 +4,7 @@
   "docs": "https://devblogs.microsoft.com/typescript/announcing-typescript-5-8-beta/#the---erasablesyntaxonly-option",
   "_version": "23.6.0",
   "compilerOptions": {
+    "rewriteRelativeImportExtensions": true,
```

Sorry for the previous suggestion, I think it's fine to include `allowImportingTsExtensions`
```suggestion
    "allowImportingTsExtensions": true,
    "rewriteRelativeImportExtensions": true,
```